app/main/views/frameworks.py

```diff
@@ -143,10 +145,40 @@ def download_supplier_pack():
 @main.route('/frameworks/g-cloud-7/updates', methods=['GET'])
 @login_required
 @flask_featureflags.is_active_feature('GCLOUD7_OPEN')
-def framework_updates():
+def framework_updates(error_message=None):
     current_app.logger.info("g7updates.viewed: user_id:%s supplier_id:%s",
                             current_user.email_address, current_user.supplier_id)
-    return _framework_updates_page()
+
+    template_data = main.config['BASE_TEMPLATE_DATA']
+
+    uploader = s3.S3(current_app.config['DM_G7_DRAFT_DOCUMENTS_BUCKET'])
+    file_list = uploader.list('g-cloud-7-updates/')
+
+    sections = [
+        {
+            'section': 'communications',
+            'heading': "G-Cloud 7 communications",
+            'empty_message': "No communications have been sent out",
+            'files': []
+        },
+        {
+            'section': 'clarifications',
+            'heading': "G-Cloud 7 clarification questions and answers",
+            'empty_message': "No clarification questions exist",
```

Comment:
Should this be "No clarification *answers* exist"?  Because some questions might have been asked but not yet answered - e.g. if I submitted a clarification question I might be worried if the message still said "no questions exist", and wonder if my message had been sent in.